.jenkins/Jenkinsfile

```diff
@@ -249,7 +249,25 @@ pipeline {
             }
           }
         }
+        stage('Coffeelake Container SGX1-FLC RelWithDebInfo') {
+          agent {
+            docker {
+              image 'oetools-azure:1.7'
+              label 'hardware'
+              args '--privileged -v /dev/sgx:/dev/sgx'
+            }
+          }
+
+          steps {
+            timeout(15) {
+              sh 'ls -l /dev'
```

Comment:
You can list out only /dev/sgx instead of all the devices.

Reply:
Oops, I should remove this. Was for debugging purposes only! Thanks Anita, nice catch